src/data/constants/menu.js

```diff
@@ -1,24 +1,22 @@
 export const MENU = [
   {
     title: 'Entradas',
-    url: '/checkout/'
+    url: '/checkout'
```

Comment:
The slashes at the end are there because gatsby adds them by default when accessing a page (either you include the slash on the url or not). If we don't do the same here, the menu doesn't display the correct active item because the url doesn't match.

Reply:
I actually removed it because the menu wasn't detecting the right one. I'll refactor this so it'll work with and without the slash.

Reply:
Done; when accessing the location, I removed `/\/$/` so we won't have to worry whether we the user accesses without the slash or the clicks on a Gastby link.

Reply:
:+1: